src/main/webapp/js/email-browser.js

```diff
@@ -420,61 +604,101 @@ var jasig = jasig || {};
 	                type: "POST",
 	                data: {} ,
 	                dataType: "json",
-	                success: function(response){  	                	
+                        success: function(response){  	                	
 		                var select = that.locate("allFolders");	
 		                $("option", select).remove();
-		                if(select.prop) {
+		                if (select.prop) {
 		                	var options = select.prop('options');
 		                }
 		                else {
 		                	 var options = select.attr('options');
 		                }
 		                var selected;
 		                $.each(response, function(index,response) {
-		                	if(index=="selected"){
+		                	if (index=="selected"){
```

Comment:
:+1: 
:flushed:, How could I have let this in the code... 

> if ( :beer: || :sake: || :cocktail: || :tropical_drink: || :wine_glass: ) {
   goToSleep();
 } else {
   goCoding();
}